tests/internal/src/combine_static_library_test/Foo1.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Foo.cpp
+ *
+ *  Created on: May 23, 2018
```

Comment:
We don't typically include author/creation dates in blt source files.  

grepping through the code, I saw one exception.  
I assume this came from your file editor.
Would you mind removing it from here and elsewhere? (not strictly necessary if you feel strongly about it)